config.yaml

```diff
@@ -0,0 +1,37 @@
+## metrics_path: /path/for/metrics; Default: /metrics
+metrics_path: /metrics
+## probe_path: /path/for/probe; Default: /probe
+probe_path: /probe
+
+## Turn on legacy mode, where the exporter scrapes a configured endpoint and exposes the metrics on the metrics_path
+## Defaults to false when using config file
+legacy_mode: false
```

Comment:
There should be no "legacy mode". Both normal scrape and probe endpoints should be supported simultaneously.